apps/total-typescript/src/templates/exercise-template.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react'
 import ModuleLessonListHeader from 'components/module-lesson-list-header'
 import Navigation from 'components/app/navigation'
 import Layout from 'components/app/layout'
-import {VideoProvider} from 'hooks/use-mux-player'
+import {useMuxPlayer, VideoProvider} from 'hooks/use-mux-player'
```

Comment:
imported, but not used
```suggestion
import {VideoProvider} from 'hooks/use-mux-player'
```